samples/04_gis_analysts_data_scientists/temperature_forecast_using_time_series_data.ipynb

```diff
@@ -69,7 +69,7 @@
   },
```

Comment:
<ul><li>same comment. Setting the zoom level before the map is rendered does not recognize the value, but instead uses a default zoom level based on location information.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>